src/Docker.PowerShell/Cmdlets/ExportContainerImage.cs

```diff
@@ -0,0 +1,43 @@
+using System.Threading.Tasks;
+using System.Management.Automation;
+using System.Collections.Generic;
+using System.IO;
+
+namespace Docker.PowerShell.Cmdlets
+{
+    [Cmdlet(VerbsData.Export, "ContainerImage",
+            DefaultParameterSetName = CommonParameterSetNames.Default)]
+    [Alias("Save-ContainerImage")]
+    public class ExportContainerImage : ImageOperationCmdlet
+    {
+        #region Parameters
+
+        [Parameter(Position = 1,
+            Mandatory = true)]
+        [ValidateNotNullOrEmpty]
+        public string FilePath { get; set; }
+
+        #endregion
+
+        #region Overrides
+
+        protected override async Task ProcessRecordAsync()
+        {
+            List<string> names = new List<string>();
+
+            foreach (var id in ParameterResolvers.GetImageIds(Image, Id))
+            {
+                names.Add(id);
+            }
+
+            using (var fs = File.Create(FilePath))
```

Comment:
Should we do a "safe" failure here if the file exists rather than forcibly overwrite it?